### PR TITLE
Add new theme preset based on Shopify's Polaris design system

### PR DESCRIPTION
**Pull request by yurm04**
Addresses Issue https://github.com/system-ui/theme-ui/issues/115 

[Polaris](https://polaris.shopify.com/design/colors) is a great design system from Shopify so here's a very basic theme preset to use as a jumping-off point.  If there's interest I can continue to further develop this preset to more closely match Polaris's typography.

Screen Shot 2020-01-12 at 7 53 41 PM

### CODE REVIEW

File: packages/preset-polaris/src/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import base from '@theme-ui/preset-base'
+
+const text = '"San Francisco", Roboto, "Segoe UI", "Helvetica Neue", sans-serif'
```
Comment (lachlanjc) [inline packages/preset-polaris/src/index.js]: "San Francisco" isn't the correct font name. Instead:
```suggestion
const text = 'system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", sans-serif'
```

File: packages/preset-polaris/src/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+export const polaris = {
+  ...base,
+  colors: {
+    text: '#454f5b',
+    background: '#fff',
+    primary: '#5C6AC4',
```
Comment (lachlanjc) [inline packages/preset-polaris/src/index.js]: These hex codes are capitalized inconsistently :) other themes seem to be using all-lowercase

File: packages/presets/src/index.js [hunk 3]
```diff
@@ -11,3 +11,4 @@ export { default as tosh } from '@theme-ui/preset-tosh'
 export { default as bootstrap } from '@theme-ui/preset-bootstrap'
 export { default as bulma } from '@theme-ui/preset-bulma'
 export { default as tailwind } from '@theme-ui/preset-tailwind'
+export { default as polaris } from '@theme-ui/preset-polaris'
```
Comment (lachlanjc) [inline packages/presets/src/index.js]: Let's keep this list alphabetized as above—just move this above Tailwind

**Comment by lachlanjc** [comment]
FYI the deploy preview seems to have failed for unrelated reasons—Yarn couldn't download `repng`. No idea what's happening there, it's still alive: https://www.npmjs.com/package/repng

**Comment by yurm04** [comment]
thanks, @lachlanjc for the feedback! Think I addressed all of your initial suggestions

**Comment by jxnblk** [comment]
Looks great! Thanks for putting this together!

**Pull request merged**